src/logcollector/logcollector.c

```diff
@@ -985,7 +969,6 @@ int handle_file(int i, int j, __attribute__((unused)) int do_fseek, int do_log)
     if (GetFileInformationByHandle(lf->h, &lpFileInformation) == 0) {
```

Comment:
Replace line 961 to _close(fd) based on [_open_osfhandle docs](https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/open-osfhandle?view=msvc-160)

Reply:
Same comment for the reload_file function behavior